DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1003 @@ Live discussion in #document-authoring on Slack.
+</figure>
+
+`@presentationTimeOffset` is a point on a sample timeline which SHALL be considered equivalent to the [=period=] start point on the [=MPD timeline=]. The value is provided by `SegmentTemplate@presentationTimeOffset` or `SegmentBase@presentationTimeOffset`, depending on the [=addressing mode=], and has a default value of 0 `@timescale` units.
+
+The relationship between a sample timeline and the [=MPD timeline=] is fully described by the combination of `@presentationTimeOffset`, `@timescale` and `Period@start`.
+
+Note: To transform a [=sample timeline=] position `SampleTime` to an [=MPD timeline=] position, use the formula `MpdTime = Period@start + (SampleTime - @presentationTimeOffset) / @timescale`.
+
+`@presentationTimeOffset` SHALL NOT change between updates of a dynamic MPD.
+
+## Media segments ## {#timing-mediasegment}
+
+A <dfn>media segment</dfn> is an HTTP-addressable data structure that contains one or more media samples.
+
+[=Media segments=] SHALL contain one or more consecutive media samples. Consecutive [=media segments=] in the same [=representation=] SHALL contain consecutive media samples.
+
+[=Media segments=] SHALL contain the media samples that exactly match the time span on the [=sample timeline=] that is assigned to the [=media segment=] by the MPD, except when using [=simple addressing=] in which case a certain amount of inaccuracy MAY be present as defined in [[#timing-addressing-inaccuracy]].
+
+Advisement: Even when using [=simple addressing=], the [=media segment=] that starts at or overlaps the [=period=] start point SHALL contain a media sample that starts at or overlaps the [=period=] start point and the [=media segment=] that ends at or overlaps the [=period=] end point SHALL contain a media sample that ends at or overlaps the [=period=] end point.
+
+## Segment addressing modes ## {#timing-addressing}
+
+This section defines the <dfn title="addressing mode">addressing modes</dfn> that can be used for referencing [=media segments=] and [=initialization segments=] in interopreable DASH presentations.
+
+Addressing modes not defined in this chapter SHALL NOT be used by DASH services. Clients SHOULD support all addressing modes defined in this chapter.
+
+You SHOULD choose the addressing mode based on the nature of the content:
+
+<dl class="switch">
+
+: Content generated on the fly
+:: Use [=explicit addressing=].
+
+: Content generated in advance of publishing
+:: Use [=explicit addressing=] or [=indexed addressing=].
+
+</dl>
+
+A service MAY use [=simple addressing=] which enables the packager logic to be very simple. This simplicity comes at a cost of reduced applicability to multi-period scenarios and reduced client compatibility.
+
+All [=representations=] in the same [=adaptation set=] SHALL use the same addressing mode. [=Representations=] in different [=adaptation sets=] MAY use different addressing modes.
```

Comment:
Consider the case where there's an overlap in which the same segment is present in both periods. Clients will probably handle this in one of two ways:

1. The client supports accurate clipping at period boundaries, but beyond that does nothing special to handle the case. Playback will be seamless, but under the hood the client probably downloaded the segment twice and applied clipping in the same way it would have done were they different segments.
1. The client is clever enough to work out that it's the same segment, and handles it in a special way to avoid downloading the segment twice or doing any clipping.

For the first case (ExoPlayer will probably fall into this bucket) the client wont care if the addressing mode changes. For the second case, I think the main question is how the client identifies that it's exactly the same segment. I'm guessing period connectivity is indicated in the manifest, but I'm not sure that's necessarily enough. Would a client also check the URLs match? If so, any addressing mode change would need to be such that the segment is still accessed with the same URL.